server/player/player.go

```diff
@@ -332,13 +333,24 @@ func (p *Player) ExecuteCommand(commandLine string) {
 	if p.Dead() {
 		return
 	}
-	args := strings.Split(commandLine, " ")
-
-	name, ok := strings.CutPrefix(args[0], "/")
+	commandLine, ok := strings.CutPrefix(commandLine, "/")
 	if !ok {
 		return
 	}
 
+	args := strings.Split(commandLine, " ")
+
+	var (
+		name  string
```

Comment:
Would be good to leave a comment here explaining this is to allow something like `/    command` to work